test/files/pos/t12878/T.java

```diff
@@ -0,0 +1,11 @@
+//> using jvm 21+
+//> using javacOpt --enable-preview --release 21
+
+int k = 0;
+// This should work as compact units have an implicit `import module java.base`, but that doesn't seem to be the case with Java 21 preview
```

Comment:
could use 25 directly, as no one cares about regression on old preview

Reply:
I recognize you might wish to see it run and pass today, however.

Reply:
Right, though we run 25 already on CI. I'll do that.